changelog

```diff
@@ -1,3 +1,61 @@
+Version 0.10.14
+  * Breaking changes:
+    * Portrait image files have been removed. (@tibetiroka)
+  * Bug fixes:
+    * Content bugs:
+      * Typo fixes. (@jmelesky, @M68HC060, @mOctave, @Saugia, @tekker2234, @tibetiroka)
+      * "Deep: Project Hawking" can no longer be offered while either of "Deep: TMBR: 0" or "Deep: TMBR: 1" is active. (@tekker2234)
+      * The Hai "Baellie" and "Basrem" engines now draw their flare sprites at 0.8 scale to differentiate them from the "Benga" engines. (@Amazinite)
+      * Fixed inconsistencies in the name of the language condition used by the Incipias governments. (@TomGoodIdea)
+      * Corrected the "index" value of the "Nelmeb GP Hybrid Thruster". (@qxxt)
+      * The Patir Mystery 12.2B and Aberrant first contact missions now use the "<flagship>" text replacement in order to refer to the player's flagship, instead of using the name of the boarded ship. (@Amazinite, @TomGoodIdea)
+      * The "Wanderers: Diplomacy" conversation now includes a branch for if the player used the Hai start. (@Amazinite)
+      * Patir 2 missions now have "on visit" messages. (@Amazinite)
+      * The Patir 2 "asteroid" ships are no longer capturable. (@Amazinite)
+      * Fixed an issue where a dialog was appearing out of order in "Remnant: Cognizance 18". (@warp-core)
+      * "Wanderers: Alpha Surveillance H" now has a very high offer precedence nad so will offer before almost any other mission. (@Amazinite)
+      * Removed references to the removed "Cutthroat (Jousting)" variant of the Cutthroat. (@Saugia)
+    * Engine bugs:
+      * Haze is now drawn at the correct size. (@palant)
+      * Changes to the starfield zoom preference will now immediately update the background when starfield animation is disabled. (@Amazinite)
+      * Planet and spaceport descriptions no longer render outside of the text box. (@TomGoodIdea)
+      * Improved map panel performance in some situations by caching mission deadline related calculations. (@Amazinite, @TomGoodIdea)
+      * Fixed various problems with text truncation. (@SomeTroglodyte)
+      * Fixed free cargo and bunks being pluralized when there is only one unit available. (@TomGoodIdea)
+      * Corrected the positioning of projectile die effects. (@Amazinite)
+  * Game content:
+    * Sprites:
```

Comment:
```suggestion
    * Missions:
      * The mission "FW Southern Recon 1B" can now be completed immediately if you already have an electron beam with you to give to Freya, instead of requiring you to take off and land again to progress the story. (@Amazinite)
    * Sprites:
```